tests/test_client.py

```diff
@@ -0,0 +1,45 @@
+import socket
+import xmlrpc.client
+
+import pytest
+from megamock import Mega, MegaMock, MegaPatch
+
+from pytest_hot_reloading.client import PytestClient
+
+
+class TestPytestClient:
```

Comment:
This stuff works, its more about testing the megamock + hot reloading experience. I will say I didn't encounter any errors about the AST being modified, which I swear I've encountered before.